controllers/component_build_controller_pac.go

```diff
@@ -984,6 +984,17 @@ func (r *ComponentBuildReconciler) UnconfigureRepositoryForPaC(ctx context.Conte
 		return baseBranch, "", "", err
 	}
 
+	action_done := "close"
+	// Close merge request.
+	// To close a merge request it's enough to delete the branch.
+
+	// Non-existing source branch should not be an error, just ignore it,
+	// but other errors should be handled.
+	if _, err := gitClient.DeleteBranch(repoUrl, sourceBranch); err != nil {
+		return baseBranch, prUrl, action_done, err
+	}
+	log.Info(fmt.Sprintf("pull request source branch %s is deleted", sourceBranch), l.Action, l.ActionDelete)
```

Comment:
I'd stress, that it's PaC configuration proposal branch

Reply:
chaned